pkg/sql/logictest/testdata/logic_test/needed_columns

```diff
@@ -67,7 +67,7 @@ query ITTTTT
 EXPLAIN (METADATA) SELECT 1 FROM (SELECT 1 AS s) WITH ORDINALITY
 ----
 0  render      ·  ·  ("1")                       ="1"
-1  ordinality  ·  ·  (s[omitted], "ordinality")  +"ordinality",unique
+1  ordinality  ·  ·  (s[omitted], "ordinality")  ·
```

Comment:
Now that you can represent multiple orderings, why not use that for ordinalityNode too then?